src/corehost/cli/deps_format.cpp

```diff
@@ -36,11 +36,20 @@ pal::string_t deps_json_t::get_optional_path(
     if (iter != properties.end())
     {
         path = iter->second.as_string();
+    }
 
-        if (_X('/') != DIR_SEPARATOR)
-        {
-            replace_char(&path, _X('/'), DIR_SEPARATOR);
-        }
+    return path;
```

Comment:
(nit) it seems like `path` is not a great name for this variable.  How about `value`?